scripts/exglobal_atmos_analysis.sh

```diff
@@ -723,7 +722,7 @@ cat > gsiparm.anl << EOF
   diag_precon=.true.,step_start=1.e-3,emiss_bc=.true.,nhr_obsbin=${nhr_obsbin:-3},
   cwoption=3,imp_physics=${imp_physics},lupp=${lupp},cnvw_option=${cnvw_option},cao_check=${cao_check},
   netcdf_diag=${netcdf_diag},binary_diag=${binary_diag},
-  lobsdiag_forenkf=${lobsdiag_forenkf},
+  lobsdiag_forenkf=.true.,
```

Comment:
On inspection, this should be reverted:

```suggestion
  lobsdiag_forenkf=${lobsdiag_forenkf}
```